vulcanai/tests/models/test_cnn.py

```diff
@@ -217,3 +217,27 @@ def test_forward_pass_class_not_nan_single_value(self,
             data_loader=test_dataloader,
             transform_outputs=False)
         assert np.any(~np.isnan(raw_output))
+
+    def test_early_stopping(self, conv3D_net_class_early_stopping,
+                            conv3D_net_class):
+        """ Test that their final params are different: aka
+        that the early stopping did something"""
+
+        ds = DataLoader(TensorDataset(
+            torch.rand(size=[10, *conv3D_net_class_early_stopping.in_dim]),
+            torch.tensor([0, 1, 2, 3, 4, 5, 6, 7, 8, 9]).long()))
+
+        conv3D_net_class.fit(
+            train_loader=ds,
+            val_loader=ds,
+            epochs=5)
+
+        conv3D_net_class_early_stopping.fit(
+            train_loader=ds,
+            val_loader=ds,
+            epochs=5)
+
+        stopping_params = list(conv3D_net_class_early_stopping.parameters())[-1].data
```

Comment:
is setting the torch seed enough or should we use our global seed to ensure they train in exactly the same fashion, otherwise this test will always pass regardless if early stopping works. Same goes for the DNN test